src/game-config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"events": {
+		"pool": [
+			"BoomerGregorEvent", "CuteGirlEvent", "FratPartyEvent"
+		],
+		"followUp": [
```

Comment:
Why call this `followUp`?

Reply:
cuz that's what it's called before. probably need to refactor to something easier to understand, such as `startUp`.

Reply:
This was just called `queue` before wasn't it?

Reply:
Yes